app/controllers/twitter_sign_in.js

```diff
@@ -4,7 +4,7 @@ var oauth = require('../../lib/oauth.js')
 var logger = require('../../lib/logger.js')()
 
 var oauthRequestTokenHandler = function (req, res) {
-  return function (err, oauth_token, oauth_token_secret, results) {
```

Comment:
It did for me, and it seems to be because I upgraded Standard to the latest version and they had some new rules, including one for "Require camelCase": http://standardjs.com/changelog.html#new-rules

Did it not raise a complaint for you?